docs/tutorial/deployments.md

```diff
@@ -14,174 +12,140 @@ search:
 ---
 # Deploying Flows
 
-## Why Deploy?
+!!! note "Reminder to start a Prefect API"
+    Some features in this tutorial such as scheduling require a Prefect server to be running; if using a self-hosted setup, simply run `prefect server start` to run both the webserver and UI; if using Prefect Cloud, make sure you have [successfully configured your local profile](/cloud/cloud-quickstart/).
 
-One of the most common reasons to use a tool like Prefect is [scheduling](/concepts/schedules). You want your flows running on production infrastructure in a consistent and predictable way. Up to this point, we’ve demonstrated running Prefect flows as scripts, but this means *you* have been the one triggering flow runs. In order to schedule flow runs or trigger them based on [events](/cloud/events/) or certain conditions, you’ll need to [deploy](/concepts/deployments/) them.
+## Why deployments?
 
-A deployed flow can be interacted with in additional ways:
+One of the most common reasons to use a tool like Prefect is [scheduling](/concepts/schedules) or [event-based triggering](/concepts/automations/). Up to this point, we’ve demonstrated running Prefect flows as scripts, but this means *you* have been the one triggering and managing flow runs. You can certainly continue to trigger your workflows in this way and use Prefect as a monitoring layer for other schedulers or systems, but you will miss out on many of the other benefits and features that Prefect offers.
 
-- you can trigger runs on remote + configurable infrastructure
-- you can trigger runs via schedules, the UI, Webhooks/Automations, or the REST API over HTTP
-- you can trigger runs with custom parameters via any of the above methods
+A deployed flow enhances a normal flow in many ways:
 
-## What is a Deployment?
+- a deployment has an API for triggering work, cancelling active runs, pausing scheduled work, customizing parameters, and more
+- you can remotely configure schedules and automation rules for your deployments
+- you can even use Prefect to dynamically provision infrastructure using [workers](/tutorials/workers/)
 
-Deploying a flow is the act of specifying where, when, and how it will run. This information is encapsulated and sent to Prefect as a [Deployment](/concepts/deployments/) which contains the crucial metadata needed for orchestration. Deployments elevate workflows from functions that you call manually to API-managed entities.
+## What is a deployment?
+
+Deploying a flow is the act of specifying when, where, and how it will run. This information is encapsulated and sent to Prefect as a [Deployment](/concepts/deployments/) which contains the crucial metadata needed for remote orchestration. Deployments elevate workflows from functions that you call manually to API-managed entities.
```

Comment:
```suggestion
Deploying a flow is the act of specifying when, where, and how it will run. This information is encapsulated and sent to Prefect as a [deployment](/concepts/deployments/) which contains the crucial metadata needed for remote orchestration. Deployments elevate workflows from functions that you call manually to API-managed entities.
```